.github/workflows/synchronize-go-mod.yml

```diff
@@ -0,0 +1,48 @@
+name: Synchronize go.mod in submodules
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - synchronize
+
+permissions:
+  contents: write
+
+jobs:
+  synchronize-go-mod:
+    if: github.actor == 'dependabot[bot]'
```

Comment:
Should this be an ID? If someone names their account this string, this would run. Not really a big deal though

Reply:
This is the [documented way](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#responding-to-events). The actor name is a user name of the entity that triggered the workflow.

Reply:
I agree with Hayden's question here - having this pinned to an immutable ID feels marginally safer.